kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -110,15 +110,8 @@ define([
             PR.prettyPrint(null, container);
 
             places = {
-                field: document.getElementById(fieldId),
-                message: document.getElementById(rendered.places.message),
-                inputControl: document.getElementById(rendered.places.inputControl),
+                inputControl: container.querySelector('#' + rendered.ids.inputControl),
```

Comment:
Looking closely at this code, it turns out that `rendered.ids.message` and `fieldId` were not set on any HTML element, and that the `infoPanel` mentioned in the code ^^^ is also unset. I've removed the unused stuff and put a comment on the `infoPanel` code to say that it is not set.